### ISSUE TITLE
Do not create `/etc/openhab/automation/jsr223` on boot

### ISSUE THREAD

**Issue by dilyanpalauzov** [openhab/openhab-core#3171]
I use openhab-js, I do not use JSR233 scripting.  When I want to edit a script under `/etc/openhab/automation/js/` I use the completions functions of the shell/editor.  These completions functions work best, if `/etc/openhab/automation/` contains a single entry (directory).  But it has two: `/etc/openhab/automation/js` and `/etc/openhab/automation/jsr233`.  Even if I delete `/etc/openhab/automation/jsr223`, it is recreated the next time openHAB starts.

Eventually I ended up with a rule:
```
rule "System started"
when
    System started
then
    executeCommandLine("/bin/bash", "-c", "/usr/bin/rmdir /etc/openhab/automation/jsr223")
end
```

Please alter openHAB not to create the directory `/etc/openhab/automation/jsr223` on start up.

I am using openHAB 3.3.

**Comment by rkoshak** [comment]
The problem is OH 3 ships with Nashorn JS and in fact you cannot remove Nashorn JS. Nashorn JS is a JSR223 rules language and requires that folder to be there. Until OH moves to a new Java version (i.e. OH 4) that folder needs to be there  for those using Nashorn Java Script. 

Removing it like you are doing may work until you import a rule as a template that uses Nashorn or create a rule using Blockly at which point errors will likely occur.

**Comment by dilyanpalauzov** [comment]
Yes, but I do not create rules with Blockly or template-rules with Nashorn.  So this empty folder shall not be re-created on each start.

**Comment by rkoshak** [comment]
It's required by parts of OH that you cannot uninstall. Whether or not you use it, it's part of OH. If OH sees something wrong (i.e. a folder required by something that is part of itself doesn't exist) it is going to recreate it. OH also resets the permissions and ownership of those folders on restart too along with some other self check/correction activities. 

If it bothers you, until OH 4 you can create a symlink to your automation folder somewhere and use that. As long as Nashorn is part of OH though, that jsr223 folder needs to be there.

**Comment by dilyanpalauzov** [comment]
I am not happy with the answer, but I do not think anything is going to change for openHAB 3.4.  Therefore I close the case.
